Servers/UI/OJS.Servers.Ui/ClientApp/src/components/administration/contests/AdministrationContestPage.tsx

```diff
@@ -20,41 +22,55 @@ const AdministrationContestPage = () => {
     const [ , , , contestId ] = pathname.split('/');
     const [ tabName, setTabName ] = useState(CONTEST_LISTED_DATA.PROBLEMS);
 
+    const [ errorMessages, setErrorMessages ] = useState <Array<string>>([]);
+
     const onTabChange = (event: React.SyntheticEvent, newValue: CONTEST_LISTED_DATA) => {
         setTabName(newValue);
     };
 
-    const { refetch: retake, data, isFetching, isLoading } = useGetContestByIdQuery({ id: Number(contestId) });
+    const { refetch: retake, data, isFetching, isLoading, error } = useGetContestByIdQuery({ id: Number(contestId) });
+
+    const { data: activityData, error: activityError, isLoading: isGettingActivity, isFetching: isFetchingActivity } =
+    useGetContestActivityQuery(Number(contestId));
+
+    useEffect(() => {
+        getAndSetExceptionMessage([ activityError, error ], setErrorMessages);
+    }, [ activityError, error ]);
 
     const renderContestEdit = () => (
         <ContestEdit contestId={Number(contestId)} currentContest={data} onSuccess={retake} />
     );
+
     const renderProblemsInContestView = (key:string) => (
         <ProblemsInContestView
           key={key}
           contestId={Number(contestId)}
           contestType={ContestVariation[data?.type as keyof typeof ContestVariation]}
+          canContestBeCompeted={activityData?.canBeCompeted || false}
         />
     );
 
     const renderParticipantsInContestView = (key: string) => (
         <ParticipantsInContestView key={key} contestId={Number(contestId)} contestName={data!.name} />
     );
 
-    if (isFetching || isLoading) {
+    if (isFetching || isLoading || isGettingActivity || isFetchingActivity) {
```

Comment:
isGetting and isFetching are synonymous to me and is confusing.

Reply:
in RTK isLoading refers to the first time when the query is executed while isFetching refers to every other time

Reply:
Just `isGetting` to `isLoading` to be consistent with RTK convention

Reply:
Fixed.